### ISSUE TITLE
Devices offline

### ISSUE THREAD

**Issue by Theetjuh** [RobertD502/home-assistant-petkit#28]
Hi,

I just found out all my devices are offline and not updating anymore.
Enabled debugging, nothing really to be found.
Reloaded the integration and restarted HA (2023.7.3) already.

The app is still working without issues and devices report as online.

Maybe others have the same issues?

**Comment by Theetjuh** [comment]
Yeah I got exactly the same

**Comment by Theetjuh** [comment]
![image](https://github.com/RobertD502/home-assistant-petkit/assets/16882588/93b47cdc-4b51-4e65-a916-dd1938a8c0fe)

07:05 was when I needed to reconfigure my account it seems, like you see of the purified water graph, it hasn't been updated since.

**Comment by RobertD502** [comment]
What about the Status sensor? What is it reporting?

**Comment by RobertD502** [comment]
Ah look at the `Status` sensor entity....your feeder is offline.

**Comment by Theetjuh** [comment]
It isn't I can operate it through the app

**Comment by marce14-git** [comment]
> It isn't I can operate it through the app

same for me HA saying offline, App works


Issues started after HA said I have to reconfigure the Integration and login again, ca 5-6 hours ago

**Comment by Theetjuh** [comment]
> > It isn't I can operate it through the app
> 
> same for me HA saying offline, App works
> 
> Issues started after HA said I have to reconfigure the Integration and login again, ca 5-6 hours ago

Exactly the same for me

**Comment by RobertD502** [comment]
Enable debugging for the integration, restart Home Assistant, wait a few minutes, then disable debug logging for the integration - a file should be downloaded for you then. Send the log to me via Discord since the data may have identifiable info and I wouldn't want you sharing that here publicly.

**Comment by Theetjuh** [comment]
@marce14-git what is the region you use within the app?

**Comment by marce14-git** [comment]
> @marce14-git what is the region you use within the app?

yeah we figured out that it is probably because the EU endpoint changed (I am using Austria)

**Comment by Theetjuh** [comment]
Aahh you where in talks as well ... sweet!

**Comment by alexpelli** [comment]
I'm facing more troubles since I have removed the HA app and now I'm not anymore able to connect the device. Everytime I get error on login and password.. no way to add again the integration

**Comment by marce14-git** [comment]
> I'm facing more troubles since I have removed the HA app and now I'm not anymore able to connect the device. Everytime I get error on login and password.. no way to add again the integration

yeah a fix is being worked on, i have the same problem now

**Comment by alexpelli** [comment]
For your info I have changed country from Italy to United states and integration get the account but I'm not able to operate it

**Comment by RobertD502** [comment]
Update for anyone seeing this: Petkit has added a new base URL for accounts they deem need to use the new EU base url. Although these accounts can still use the current base url, the devices will report as offline. They only report as being online when the new base url is being used.

I am currently working on the fix.
